### PR TITLE
Add basic ros architecture

### PR DESCRIPTION
**Pull request by jcnorby**
Should include all core nodes and topics, with correct message types

### CODE REVIEW

File: spirit_msgs/msg/ControlInput.msg [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# This is a message to hold a spirit control input
+#
+# The control input is defined as the joint positions, joint velocities, body position, body velocity, body orientation, body angular velocity
```
Comment (nikolaif399) [inline spirit_msgs/msg/ControlInput.msg:3]: Is this comment correct? What gets stored in control input seems to differ in the below message declaration.

**Review by nikolaif399 — APPROVED** [review]

File: spirit_msgs/msg/ControlInput.msg (hunk 1 above)
Comment (jcnorby) [reply spirit_msgs/msg/ControlInput.msg:3]: @Kongx231

Comment (Kongx231) [reply spirit_msgs/msg/ControlInput.msg:3]: That is not correct. I think I accidentally kept the state estimation comment in there. I think it should be the cartesian force in the world frame for each leg

Comment (jcnorby) [reply spirit_msgs/msg/ControlInput.msg:3]: You should just be able to make the change and push it

Comment (nikolaif399) [reply spirit_msgs/msg/ControlInput.msg:3]: Soon as this gets modified feel free to pull in to master @jcnorby

Comment (jcnorby) [reply spirit_msgs/msg/ControlInput.msg]: Never mind, changed.

**Pull request merged**